lib/rspec-puppet/support.rb

```diff
@@ -228,9 +228,15 @@ def build_catalog(*args)
     end
 
     def munge_facts(facts)
-      output = {}
-      facts.keys.each { |key| output[key.to_s] = facts[key] }
-      output
+      return facts.reduce({}) do | memo, (k, v)|
```

Comment:
David:  I want to be sure that this is all right given your comment below about using a string literal rather than the symbol in the fact.  I don't have an understanding for why rspec-puppet requires the use of string literals in nested data structures--I know that facter does not support the use of symbols, but have often thought that rspec-puppet should convert symbols as a matter of convenience.